src/data/meetings.json

```
@@ -122,11 +122,27 @@
     "title": "Ad Hoc Meeting Space",
     "localVideo": {},
     "remoteVideo": null,
-    "localAudio": {},
+    "localAudio": {
+      "muted": false
+    },
     "remoteAudio": null,
     "localShare": null,
     "remoteShare": null
   },
+  "localMediaWithSharing": {
+    "ID": "localMediaWithSharing",
+    "title": "Ad Hoc Meeting Space",
+    "localVideo": {
```

Since this is supposed to represent a meeting with screen share, the attributes that represent a display media stream should not be `null`, and instead be an object to indicate that "they exist". Like so:
```suggestion
    "localVideo": {},
    "remoteVideo": {},
    "localAudio": {},
    "remoteAudio": {},
    "localShare": {},
    "remoteShare": null
```

In this case, having an object in the `localShare` attribute indicates that the user is sharing their screen. Conversely, if `remoteShare` was an object `{}` instead of `null`, while `localShare` is `null`, this would represent a meeting where a remote participant is sharing their screen.

---

How about to set both the `localShare` and the `remoteShare` to `null`?  This scenario requires the user to click on the sharing button by themself. The sharing video isn't acquired automatically on starting.

---

I'm ok with that, if it's not necessary to add the share stream. In that case, I would look to the existing objects to see if you can reuse any of them instead of adding new one

---

good idea. Will reuse "remote&localMedia" instead of the new one.